docs/canvas/canvas-share-workpad.asciidoc

```diff
@@ -2,47 +2,94 @@
 
 . From your workpad, click the *Share workpad* icon in the upper left corner, then select *PDF reports*.
 
-. Click *Generate PDF*. 
+. Click *Generate PDF*.
 +
 [role="screenshot"]
 image::images/canvas-generate-pdf.gif[Generate PDF]
 
 [float]
-[[export-workpad]]
-=== Export your workpad
+[[create-workpad-URL]]
+=== Create a POST URL
 
-To share your workpad with another author, export it as a JSON file.
+Create the POST URL that automatically generates PDF reports with Watcher or a script.
+
+For more information, refer to <<automating-report-generation, Automating report generation>>.
+
+. If you are using a Gold or Platinum license, enable reporting in your `config/kibana.yml` file.
+
+. From your workpad, click the *Share workpad* icon in the upper left corner, then select *PDF reports*.
+
+. Click *Copy POST URL*.
++
+[role="screenshot"]
+image::images/canvas-create-URL.gif[Create POST URL]
 
 [float]
-[[export-single-workpad]]
-==== Export a single workpad
+[[add-workpad-website]]
+=== Share the workpad on a website
 
-. From your workpad, click the *Share workpad* icon in the upper left corner.
+beta[] To show off your workpad outside of {kib}, share a static version of the workpad on an external website.
 
-. Select *Download as JSON*.
+To make sure that your data remains secure, the data in the JSON file is not connected to {kib}. Canvas does not display elements that manipulate the data on the workpad.
+
+. If you are using a Gold or Platinum license, enable reporting in your `config/kibana.yml` file.
+
+. From your workpad, click the *Share this workpad* icon in the upper left corner, then select *Share on a website*.
+
+. On the *Share on a website* pane, follow the instructions.
 +
 [role="screenshot"]
-image::images/canvas-export-workpad.png[Export single workpad]
+image::images/canvas-embed_workpad.gif[Share the workpad on a website]
+
+NOTE: Shareable workpads encode the current state of the workpad in the JSON file. When you make changes to the workpad, the changes do not appear in the shareable workpad on your website.
 
 [float]
-[[export-multiple-workpads]]
-==== Export multiple workpads
+[[shareable-workpad-enable-autoplay]]
+==== Enable autoplay
+
+Automatically cycle through the workpads pages.
+
+. In the lower right corner of the shareable workpad, click the settings icon.
 
-. Go to the *Canvas workpads* page.
+. Click *Auto Play*, then select *Cycle Slides*.
+
+. In the *Set a custom interval* field, enter the interval you want to use, then click *Set*.
+
+. To use the workpad animations, select *Animate cycle*.
+
+[float]
+[[hide-workpad-toolbar]]
+==== Hide the toolbar
 
-. Select the workpads you want to export
+The toolbar displays the number of pages in your workpad, and the settings icon. If you don't want the toolbar to appear, you can hide it.
 
-. Click *Export*.
+. In the lower right corner, click the settings icon.
 
+. Click *Toolbar*, then select *Hide Toolbar*.
```

Comment:
Can we add a note that these can be set up to be enabled in the code, see the instructions for details?

Reply:
or can we add a table of configuration options to this document as well?